TestRunner.psm1

```diff
@@ -499,48 +497,55 @@ function Invoke-TestRunner {
             $outputDir = Resolve-Path $testKitOutputPath
             Push-Location ./src/edfi-paging-test
 
-            $params = @(
-                "run",
-                "python",
-                "edfi_paging_test",
-                "--baseUrl", $url,
-                "--key", $key,
-                "--secret", $secret,
-                "--output", $outputDir
-            )
-            if ($clientCount) {
-                $params += @("--connectionLimit", $clientCount)
-            }
-            if ($contentType) {
-                $params += @("--contentType", $contentType)
-            }
-            if ($resourceList) {
-                $params += @("--resourceList", $resourceList)
-            }
-            if ($pageSize) {
-                $params += @("--pageSize", $pageSize)
+            try {
+                Write-DebugLog "Executing: poetry install" -LogLevel $logLevel
+                &poetry install
+
+                $command = "poetry run python edfi_paging_test --baseUrl $url --key  $key --secret $secret --output $outputDir"
+                if ($clientCount) {
+                    $command += " --connectionLimit  $clientCount"
+                }
+                if ($contentType) {
+                    $command += " --contentType  $contentType"
+                }
+                if ($resourceList) {
+                    $command += " --resourceList $resourceList"
+                }
+                if ($pageSize) {
+                    $command += " --pageSize $pageSize"
+                }
+                if ($logLevel) {
+                    $command += " --logLevel $logLevel"
+                }
+                if ($description) {
+                    $command += " --description '$description'"
+                }
+
+                Write-DebugLog "Executing: $command" -LogLevel $logLevel
+                Invoke-Expression -Command $command
```

Comment:
Changing to `Invoke-Expression` instead of `Start-Process` (old line 530 below) means that the output goes right to the console and is included in the transcript logging.